magenta/music/midi_synth.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MIDI audio synthesis."""
+
+# internal imports
+import numpy as np
+
+from magenta.music import midi_io
+
+
+def synthesize(sequence, sample_rate, wave=np.sin):
+  """Synthesizes audio from a music_pb2.NoteSequence using a waveform.
```

Comment:
We should probably mention that the quality will be lower than using fluidsynth with a good soundfont.